rust/cli/src/convert.rs

```diff
@@ -26,6 +26,8 @@ pub struct Cli {
     ///
     /// Defaults to inferring the format from the file name extension
     /// of the `input`.
+    ///
+    /// To view all supported file formats uses the codecs command.
```

Comment:
```suggestion
    /// Defaults to inferring the format from the file name extension of the `input`.
    /// See `stencila codecs list` for available formats.
```

Reply:
Made some suggestions here:

- do not use a new paragraph to avoid unnecessary vertical spacing in `--help`
- use the full command and place it in backticks to make it clearer what needs to be run (we do this elsewhere too)

For consistency, this sentence should be added to all the `--from` and `--to` options in all the relevant commands `convert`, `compile`, `execute`, and `render`.